src/_guides/libraries/library-tour.md

```diff
@@ -1176,15 +1183,15 @@ The equivalent code with await expressions,
 including an asynchronous for loop (`await for`),
 looks more like synchronous code:
 
-<!-- dart-tutorials-samples/cmdline/bin/dgrep.dart -->
+<?code-excerpt "lib/library_tour/async/stream.dart (await-for)" replace="/await for/[!$&!]/g"?>
 {% prettify dart %}
-main(List<String> arguments) async {
-  ...
+Future main(List<String> arguments) async {
```

Comment:
I wonder if we can elide the return type `Future` in this section, just to keep the code at a minimum.

Reply:
While we elided return type annotations for `main()` functions at the start of the Language Tour to keep the code as light as possible, IMHO, our Library Tour should illustrate the recommended practice of specifying the return type of public APIs.

Generally speaking, I find it easy to gloss over the "async" keyword, whereas the "Future" annotation at the start of the method signature is usually grabs one attention :).

I'd vote to keep it. If you'd still like for me to remove it, let me know.

Reply:
I'd prefer to remove the type before main(). It doesn't seem to help, and it obscures the method name.